strings-csharp/RemoveWhitespaceCharactersFromString/RemoveWhitespaceCharactersFromString/RemoveWhitespaceMethods.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+
+namespace RemoveWhitespaceCharactersFromString
+{
+    public class RemoveWhitespaceMethods
+    {
+        public static string RemoveWhitespacesUsingRegex(string source)
+        {
+            return Regex.Replace(source, @"\s+", String.Empty);
+        }
+
+        public static string RemoveWhitespacesUsingLinq(string source)
+        {
+            return string.Concat(source.Where(c => !char.IsWhiteSpace(c)));
+        }
+
+        public static string RemoveWhitespacesUsingReplace(string source)
+        {
+            return source.Replace(" ", string.Empty);
+        }
+
+        public static string RemoveWhitespacesUsingTrim(string source)
+        {
+            return source.Trim();
```

Comment:
The method name is ok but can we improve it clearer and say `RemoveLeadingAndTrailingWhitespacesUsingTrim()`?